src/main/java/com/epam/test_generator/services/utils/UtilsService.java

```diff
@@ -109,8 +109,8 @@ public static void tagBelongsToCase(Tag tag, Case caze) {
         Set<Tag> tagSet = caze.getTags();
```

Comment:
Is this method still used?

Reply:
No. There are 6 more methods in UtilsService which are not used. Shoud we remove them completely?

Reply:
Done. Removed all unused methods in UtilsService.